elastic/eui
# [CHORE] Removing last Jenkins blocks from docs deploy script.

## Summary

Now that our Buildkite job is running correctly on production docs deploy jobs, I'm removing the last of Jenkins-specific code blocks from the `deploy_docs.sh` script.

## QA

QA will be done manually. This will include verifying the `echo` messages and `ENV` variables we expect to see in Buildkite to confirm logic is working as it was before the Jenkins code was removed.
